src/components/Sidebar.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import {SideRow} from "./SideRow";
+import HistoryIcon from "@material-ui/icons/History";
+import HomeIcon from "@material-ui/icons/Home";
+import ThumbUpAltIcon from "@material-ui/icons/ThumbUpAlt";
+import VideoLibraryIcon from "@material-ui/icons/VideoLibrary";
+import "./Sidebar.css";
+
+export function Sidebar({setShowSidebar}) {
+  return (
+    <div
+      className="card-shadow-1 sidebar-container"
+      style={{
+        backgroundColor: "var(--sidebar-color)",
+        color: "var(--text-color)",
+        minHeight: "100vh",
+        width: "15rem",
+      }}
+    >
+      <div className="nav-left mg-left-half lg pd-1">
+        <span
+          className="mg-left-half flex row align-items-center lg mg-bottom-1 mg-top-1"
+          style={{color: "white"}}
+        >
+          CRUNCH <span style={{color: "var(--sidebar-hover-color"}}>TUBE</span>
+        </span>
+      </div>
+
+      <SideRow title={"Home"} to="/" Icon={HomeIcon} />
```

Comment:
Extract the `<SideRow />`'s into a function which will take an array of objects and render the `SideRow` component

```suggestion
const sideRowData = [
    {
        title: 'Home',
        to: '/',
        Icon: HomeIcon
    }, {
        title: 'History',
        to: 'history',
        Icon: HistoryIcon
    }
];

const renderSideRows = ( siderows )=> 
    siderows.map({ title, to, Icon }) => (
        <SideRow title={title} to={to} Icon={Icon} />
    )
```